docs/spec/api.md

```diff
@@ -1,3 +1,7 @@
+page_title: Docker Registry HTTP API V2
```

Comment:
right now, we are displaying these specifications in the documentation. So, if I don't get this in there, the heading and the navigation breaks.  Your choice, the broken version:

![image](https://cloud.githubusercontent.com/assets/1347209/7401259/8d654698-ee77-11e4-995b-10073507e996.png)

Or the fixed version:

![image](https://cloud.githubusercontent.com/assets/1347209/7401285/b8e5b564-ee77-11e4-96c8-f689e5384401.png)